cmd/oras/internal/display/status/text.go

```diff
@@ -161,12 +162,12 @@ func (ch *TextCopyHandler) PreCopy(_ context.Context, desc ocispec.Descriptor) e
 // PostCopy implements PostCopy of CopyHandler.
 func (ch *TextCopyHandler) PostCopy(ctx context.Context, desc ocispec.Descriptor) error {
 	ch.committed.Store(desc.Digest.String(), desc.Annotations[ocispec.AnnotationTitle])
-	successors, err := graph.FilteredSuccessors(ctx, desc, ch.fetcher, DeduplicatedFilter(ch.committed))
+	deduplicated, err := graph.FilteredSuccessors(ctx, desc, ch.fetcher, DeduplicatedFilter(ch.committed))
 	if err != nil {
 		return err
 	}
-	for _, successor := range successors {
-		if err = ch.printer.PrintStatus(successor, copyPromptExists); err != nil {
+	for _, successor := range deduplicated {
+		if err = ch.printer.PrintStatus(successor, copyPromptSkipped); err != nil {
 			return err
```

Comment:
BTW, we need to create an interface for `output.Printer` otherwise it's impossible to cover this line like tty prompter.